### ISSUE TITLE
[Bug]: kdiff3 version 1.11.0 fails at first when started from command line

### ISSUE THREAD

**Issue by Taar2** [ScoopInstaller/Extras#13303]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

kdiff3

### Expected/Current Behaviour

When I call kdiff3 (version 1.11.0 obtained from extras) from the command line like:
`kdiff3 file1 file2`
I expect `kdiff3` to open the GUI and then immediately open the two files and then display them in its left and right tabs.

What I get instead is that kdiff3 opens the GUI then fails to open the files _at first_, showing two dialogs

Then after I close the dialogs, kdiff3 _manages to open the files correctly_.

When I replace manually the contents of the scoop 1.11.0 directory with that from version 1.10.7, everything works correctly.

Note that the bundle from version 1.11.0 obtained directly from kdiff3 (not going through scoop) has the same buggy behaviour

### Steps to Reproduce

```console
scoop install extras/kdiff3

kdiff3 file1.txt file.txt
```

### Possible Solution

See above. Maybe rollback to v10.7.0 until the problem is solved on the kdiff3 side?

### Scoop and Buckets Version

```console
PS> scoop --version                                                                   
Current Scoop version:                                                                            
d337bb1f (HEAD -> master, tag: v0.4.2, origin/master, origin/HEAD) chore(release): Bump to version
 0.4.2 (#5964)                                                                                    
                                                                                                  
'main' bucket:                                                                                    
561954d4d (HEAD -> master, origin/master, origin/HEAD) slang: Update to version 2024.1.16         
                                                                                                  
'extras' bucket:                                                                                  
6469c6ff0 (HEAD -> master, origin/master, origin/HEAD) sidekick-browser: Update to version 120.58.
1.38488-aa4b5ae
```

### Scoop Config

```console
PS> scoop config                                                                
                                                                                            
last_update                       scoop_repo                              scoop_branch      
-----------                       ----------                              ------------      
2024-05-16T13:58:18.1391895+02:00 https://github.com/ScoopInstaller/Scoop master
```

### PowerShell Version

```console
PS> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.19041.4291
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.4291
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by aliesbelik** [comment]
May relate to https://bugs.kde.org/show_bug.cgi?id=486782.

**Comment by Taar2** [comment]
Yes, those are the same symptoms exactly. Meanwhile, shouldn't someone rollback the manifest to @1.10.7?

This is the base url: https://download.kde.org/stable/kdiff3/kdiff3-1.10.7-windows-x86_64.exe

I can't think I'm the only one experiencing this.

**Comment by aliesbelik** [comment]
Not sure that pinning manifest to previous version is a good idea since it's only possible via removing `checkver`/`autoupdate` sections (otherwise it will be updated automatically again).

I've changed `autoupdate` template (76a0cff) so that autogeneration for old versions will work, i.e. you could install previous version via:
```
scoop install kdiff3@1.10.7
```

Alternativelly you could switch to previous version via [scoop reset](https://github.com/ScoopInstaller/Scoop/wiki/FAQ#how-do-i-switch-between-different-versions-of-an-app):
```
scoop reset kdiff3@1.10.7
```

**Comment by lucaskoliveira** [comment]
I'm getting the same behavior running 1.11 on windows 11.

**Comment by aliesbelik** [comment]
Please recheck with 1.11.1.

**Comment by lucaskoliveira** [comment]
Same error.

**Comment by Taar2** [comment]
I got *another* error with @1.11.1 on Windows 10: wrong entry point into ``...\scoop\apps\kdiff3\current\bin\KF6XmlGui.dll``. kdiff3@1.11.1 doesn't even start.

**Comment by kaesaecracker** [comment]
I do not think this is scoop specific as I am experiencing the error on the current exe installer (1.11.1) on win10.

**Comment by iq2luc** [comment]
Hello,

same on Archlinux (kdiff3 1.11.1, plasma 6.1.0, linux 6.9.6).

Console output:

```
[~/temp]> kdiff3 a.txt b.txt
org.kde.kdiff3: Loading A:  "/home/luc/temp/a.txt"
org.kde.kdiff3: Loading B:  "/home/luc/temp/b.txt"
org.kde.kdiff3: Loading A:  "/home/luc/temp/a.txt"
org.kde.kdiff3: Loading B:  "/home/luc/temp/b.txt"
org.kde.kdiff3: Diff: A <-> B
org.kde.kdiff3: Linediff: A <-> B
org.kde.kdiff3: Enter: calcDiff3LineListUsingAB
org.kde.kdiff3: Leave: calcDiff3LineListUsingAB
```

Screenshots:

Right after running:
![kdiff3_ss01](https://github.com/ScoopInstaller/Extras/assets/13350093/afa6377e-b6c4-4c29-9d52-06620c1171b8)

After clicking "OK":
![kdiff3_ss02](https://github.com/ScoopInstaller/Extras/assets/13350093/3fa8c660-e67f-4a7a-a2e3-39a8e462fc19)

After clicking "OK" again:
![kdiff3_ss03](https://github.com/ScoopInstaller/Extras/assets/13350093/36c7c78d-687a-4787-ab18-57706c0fc67c)

**Comment by tsujan** [comment]
> same on Archlinux...

Confirming. Random crashes also happen. The Qt6 port of kdiff3 is buggy; I downgraded to 1.10.7.

**Comment by LCerebo** [comment]
According to https://bugs.kde.org/show_bug.cgi?id=487893 this should be fixed in 1.11.2.
I confirm that locally, on win 11, the 1.11.2 is working.
See #13761 .

**Comment by wilbit** [comment]
> > same on Archlinux...
> 
> Confirming. Random crashes also happen. The Qt6 port of kdiff3 is buggy; I downgraded to 1.10.7.

Same on Windows 11.
In WinGet two last available versions (1.11.5 & 1.11.0) do not work for me. I downgraded to version 1.10.7 as well.
